### DISCUSSION TITLE
I need help!

### DISCUSSION THREAD

**Discussion by mariajorrat** [community/community#53740] [GitHub Education]

**Comment by samus-aran** [comment]
Hi @jorratmf ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have closed this discussion, and we ask that you no longer cross-post.

Let's continue our conversation in your [original discussion.](https://github.com/orgs/community/discussions/51808)

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Reply by mariajorrat** [reply]
I only created one post asking for help, and I did not duplicate my comment in any other discussion or forum. I apologize for any confusion that may have arisen and hope that my original post can be reinstated.

**Reply by samus-aran** [reply]
Your original discussion is still open. I've replied to it recently.

https://github.com/orgs/community/discussions/51808
